changelogs/fragments/420-release-role-poetry.yml

```
@@ -0,0 +1,4 @@
+breaking_changes:
+  - "The ``build-release`` role no longer uses poetry to run antsibull, but assumes that antsibull is installed.
+     To revert to the old behavior, set ``antsibull_build_command`` to ``poetry run antsibull``
```

In `roles/build-release/defaults/main.yaml`?

---

No, in your playbook, inventory, or passed as an extra variable. Is the following more clear?
```suggestion
     To revert to the old behavior, set the Ansible variable ``antsibull_build_command`` to ``poetry run antsibull``
```